cloud-aws-cloudformation/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/AwsMetadataCollector.java

```diff
@@ -206,4 +212,25 @@ private void addKnownInstance(CloudInstance cloudInstance, Multimap<String, Inst
         }
         return cloudLoadBalancerMetadata;
     }
+
+    @Override
+    public Map<String, Integer> collectInstanceStorageCount(AuthenticatedContext ac, List<String> instanceTypes) {
```

Comment:
Instead of Map<String, Int> can we return a class that wraps the Map<String, Int> with helper methods? That would be easier to extend in the future.

Reply:
Done